.devcontainer/on-create.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Exit on any error
+set -e
+
+echo "🛠️  Running container setup (onCreate)..."
+
+# Enable corepack for package manager management
+echo "📦 Enabling corepack for package management..."
+if sudo corepack enable; then
+  echo "✅ corepack enabled successfully"
+else
+  echo "❌ Failed to enable corepack"
+  exit 1
+fi
+
+# Install latest pnpm using corepack
+echo "📦 Installing latest pnpm using corepack..."
+if sudo corepack prepare pnpm@latest --activate; then
+  echo "✅ pnpm installed successfully via corepack"
+else
+  echo "❌ Failed to install pnpm via corepack"
+  exit 1
+fi
+
+# Install common build tools globally
+echo "🔧 Installing global build tools..."
+if sudo npm install -g turbo@latest tsup@latest typescript@latest; then
```

Comment:
I assume you are installing these globally just for convenience of writing `turbo ...` instead of `pnpm exec turbo ...`?

Doesn't this mean the versions in the root package.json will not be respected, and it would actually use a different installed copy when executing via `pnpm` versus not?

Maybe there is a way to just alias these commands to the version in `node_modules/.bin`?